Mathlib/Combinatorics/Digraph/Basic.lean

```diff
@@ -0,0 +1,236 @@
+/-
+Copyright (c) 2024 Kyle Miller, Jack Cheverton. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller, Jack Cheverton, Jeremy Tan
+-/
+import Mathlib.Order.CompleteBooleanAlgebra
+import Mathlib.Data.Fintype.Pi
+
+/-!
+# Digraphs
+
+This module defines directed graphs on a vertex type `V`. Although the structure is definitionally
+equivalent to a `Quiver` in a specialized universe, the intention is to consider graph theory
```

Comment:
It's not defeq to `Quiver` in Lean, right? Because it's a brand new structure.
I agree that it is equivalent in the `Equiv` sense.

Reply:
How about "structurally equivalent"?

The comment about "not intended to be used as a class" is true, but it might give the wrong impression. The purpose of it not being a class is to be able to work with the whole lattice of digraphs on a given vertex type. Classes work best when there's the assumption that there's *the* digraph structure on the vertex type.

The more apt comparison by the way is to a plain relation. I'd start it out with something like "This module defines directed graphs on a vertex type `V`, which is simply the notion of a relation `V -> V -> Prop`. The intention is to develop the notion using the language of graph theory." Then in the next paragraph it could mention the relation to quivers.

This will also help graph theorists understand this module, since many have never heard of quivers before.